src/test/java/sdu/sem2/se17/domain/persistenceinterface/CreditHandlerTest.java

```diff
@@ -5,45 +5,60 @@
 import sdu.sem2.se17.domain.credit.Participant;
 import sdu.sem2.se17.domain.credit.Role;
 import sdu.sem2.se17.domain.production.Production;
+import sdu.sem2.se17.persistence.data.CreditHandlerImpl;
+import sdu.sem2.se17.persistence.data.ParticipantHandlerImpl;
+import sdu.sem2.se17.persistence.db.DataSource;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class CreditHandlerTest {
 
     private CreditHandler handler;
 
+    private final boolean connectToDb = false;
+
     @BeforeEach
     void setUp() {
-        handler = new CreditHandlerImplSample();
+        if (connectToDb){
+            var ds = new DataSource("jdbc:postgresql://localhost:5432/tv2", "postgres", "postgres");
```

Comment:
Database name is set in DataSource, method expects just the server url.
```suggestion
            var ds = new DataSource("jdbc:postgresql://localhost:5432/", "postgres", "postgres");
```